src/tls/settings.rs

```diff
@@ -283,10 +283,9 @@ impl TlsSettings {
             load_mac_certs(context).unwrap();
         }
 
-        if self.alpn_protocols.is_some() {
-            let alpn = self.alpn_protocols.as_ref().unwrap();
+        if let Some(apln) = &self.alpn_protocols {
             context
-                .set_alpn_protos(alpn.as_slice())
+                .set_alpn_protos(apln.as_slice())
```

Comment:
perhaps name the variable "alpn" to avoid confusion? Which I realize now that was my suggestion 😆

Reply:
Doh! I made that misspelling many times. Thanks for catching.